README.md

```diff
@@ -225,10 +225,47 @@ You should pass this options previous initialization of the widget. Also you can
 </script>
 ```
 
+## Styled button
+If you don't want to create a button in your site, style it and link to the widget you can use our styled button that it's fully customizable.
+We are linking the button to the widget automatically!
+
+### Minimum configuration
+```js
+  <script>
+    everyMonthWidget.createButton('#yourElementToAttachTheButton')
+  </script>
+```
+### Customize the button
+The button accepts an object with the following properties to configure the styles:
```

Comment:
Eventually we can just only document this style and deprecate the DIY version, once we get to a copy-pasta-able "build your donate button" type of flow.